userspace/engine/rule_loader.cpp

```diff
@@ -750,10 +811,15 @@ static shared_ptr<ast::expr> parse_condition(
 	}
 	catch (const sinsp_exception& e)
 	{
+		libsinsp::filter::parser::pos_info pos;
+		p.get_pos(pos);
```

Comment:
I think we have a 0-params overload that allows:
```suggestion
		auto pos = p.get_pos();
```

Reply:
I created a standalone location struct and YAML::Marks and parser::pos_infos are converted now.